app/javascript/src/Onboarding.jsx

```diff
@@ -235,49 +246,52 @@ class Onboarding extends Component {
   }
 
   handleNextHover() {
-    if (this.state.pageNumber === 2 && this.state.users.length === 0) {
+    const { pageNumber, users } = this.statel;
```

Comment:
Looks like a typo here.
```suggestion
    const { pageNumber, users } = this.state;
```